<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>969</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1271</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1271</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 255, 238);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1260</width>
       <height>969</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1240</width>
          <height>560</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1220</width>
             <height>700</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="minimumSize">
               <size>
                <width>1200</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1200</width>
                <height>80</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(12, 159, 161);
border-radius : 7px;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QPushButton" name="Sc2_Home">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
QPushButton{
			
			
	image: url(:/icon/icon.svg/home.png);
			background-color: rgb(12, 159, 161);
			border-radius : 7px;
			
}

QPushButton:hover{
			
			background-color: rgb(14, 194, 194);
}

QPushButton:pressed {

			
	background-color: rgb(11, 147, 147);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>1100</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>HY헤드라인M</family>
                   <pointsize>28</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string>ND </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="minimumSize">
               <size>
                <width>1200</width>
                <height>260</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1200</width>
                <height>260</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QVideoWidget" name="Sc2_VideoR" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QVideoWidget" name="Sc2_VideoL" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Sc2_imgRx">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Sc2_imgRy">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="minimumSize">
               <size>
                <width>1200</width>
                <height>260</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1200</width>
                <height>260</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QVideoWidget" name="Sc2_DotR" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color : rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QVideoWidget" name="Sc2_DotL" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>240</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Sc2_imgLx">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Sc2_imgLy">
                 <property name="minimumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>320</width>
                   <height>240</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(12, 159, 161);</string>
        </property>
        <property name="text">
         <string>- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame_8">
        <property name="minimumSize">
         <size>
          <width>1100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="minimumSize">
            <size>
             <width>470</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>470</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTableWidget" name="Sc2_ListView">
              <property name="minimumSize">
               <size>
                <width>450</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>450</width>
                <height>300</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>D2Coding</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_10">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QPushButton" name="Sc2_Play">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
			
			
	image: url(:/icon/icon.svg/play.png);
			border-radius : 10px;
			background-color: rgb(12, 159, 161);
}

QPushButton:hover{
			
			background-color: rgb(14, 194, 194);
}

QPushButton:pressed {

			
	background-color: rgb(11, 147, 147);
}

</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Sc2_Pause">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
			
	image: url(:/icon/icon.svg/pause.png);
			border-radius : 10px;
			background-color: rgb(12, 159, 161);
}

QPushButton:hover{
			
			background-color: rgb(14, 194, 194);
}

QPushButton:pressed {

			
	background-color: rgb(11, 147, 147);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_9">
           <property name="maximumSize">
            <size>
             <width>420</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="Sc2_ResultView">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>D2Coding</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource2.qrc"/>
 </resources>
 <connections/>
</ui>
